README.md

```diff
@@ -1,111 +1,63 @@
 
-This README is the first help for anyone new to SuperCollider. There are many
-good resources available for learning SuperCollider, with up to date links from
-the site listed above, including links to forums and mailing lists.
+To get the latest stable version, Linux users will need to build SuperCollider themselves. See the [Linux README](LINUX_README.md) for instructions.
 
+[downloads page]: https://supercollider.github.io/download
 
-Usage
+Learn
 -----
 
-SuperCollider consists of three separate components:
-
-1. scsynth or supernova - audio engine (the "server")
-2. sclang - programming language runtime interpreter including Qt graphical user interfaces
-3. IDE (integrated development environment) - an editor for writing code and running supercollider
-
-To begin using SuperCollider, you usually start up the IDE:
+The official SC docs can be viewed in the SuperCollider IDE's built-in help viewer. A copy is also hosted online at [doc.sccode.org](http://doc.sccode.org/).
 
-- on Mac OS this is `SuperCollider.app`
-- on Linux and Windows, this is the `scide` executable
+We recommend the following third-party tutorials:
 
-You can get further help by using the IDE's integrated help, a system which can be invoked via the "Help" menu or using the key combination
-`Ctrl+D` (on Linux, Windows) or `Cmd+D` (on Mac OS). In the Help menu, you will also find an action that will take you directly to the help page on using the IDE.
+- [A Gentle Introduction to SuperCollider](https://works.bepress.com/bruno-ruviaro/3/), a free ebook by Bruno Ruviaro
+- [Eli Fieldsteel's video tutorials](https://www.youtube.com/playlist?list=PLPYzvS8A_rTaNDweXe6PX4CXSGq4iEWYC)
 
-When starting the SuperCollider IDE, the audio server is not started
-automatically. You can start it using the "Language > Boot Server" menu action,
-or using the key combination `Ctrl+B` (on Linux, Windows) or `Cmd+B` (on Mac OS).
-This is just to remind you in case you're impatient and can't understand why
-you're not immediately getting sound.
-
-We suggest you proceed by reading the tutorials available in the help
-system. Please note that some help pages are not up-to-date with the latest
-development of SuperCollider, especially when mentioning the code editing
-environment. Regarding this, it is best to refer to the help page specifically
-about the new SuperCollider IDE.
-
-Also, please explore the IDE menus, which will let you discover a lot of the functionality of the coding environment.
+Discuss
+-------
 
-To get further information on SuperCollider usage or development, you should subscribe
-to the mailing lists:
+You can [sign up for mailing lists][list signup], and view the archives for [sc-users][sc-users archives] and [sc-dev][sc-dev archives].
 
-http://www.birmingham.ac.uk/facilities/ea-studios/research/supercollider/mailinglist.aspx
+We also have a [forum][Discourse] and a [Slack][Slack].
```

Comment:
Prolly not worth listing here but maybe include a link to the longer list of communities/links on the github.io site? If you do, maybe better to name this section "Community"

Reply:
Also, I feel this section should have an opinion about where you might want to go for what type of discussion? I guess that's in flux lately.. thoughts?

Reply:
dunno man, i was agonizing over exactly that when writing this. decided to go with completely neutral wording that puts the lists first.

i'll add in some more communities.

Reply:
I think at the very least it would be nice to indicate which of them is the preferred place for people to discuss. but if we can't answer that question clearly right now, no point in thinking too hard about it